code-challenges.js

```diff
@@ -1,31 +1,65 @@
 // ASSESSMENT 1: Coding practical questions
-
+//
 // Please read all questions thoroughly
 // Psuedo coding is highly recommended
-
+//
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
-
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
-
+// declare a function named boilingPoint that takes a numbers (num) as an argument
+const boilingPoint = (num) => {
+  // declare an if that returns "(num) is below boiling point" if (num) is <= 211
+  if (num <= 211) {
+    return (num) + " is below boiling point."
+    // declare an else if that returns "(num) is at boiling point" if (num) === 212
+  } else if (num === 212)  {
+      return  (num) + " is at boiling point."
+      // declare an else if that returns "(num) is above boiling point" if (num) <=213
+}   else if (num >= 213) {
+  return (num) + " is above boiling point"
+  //declare an else that returns "error: please insert a number"
+}   else return "error: please insert a number"
+}
+// call function
+console.log(boilingPoint(temp1));
+console.log(boilingPoint(temp2));
+console.log(boilingPoint(temp3));
+console.log(boilingPoint("love"));
 
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
+//decalre a function that takes an array as an argument
 var myNumbers1 = [3, 7, 0, 6, -9]
-
-
+const mult5 = (array) => {
+  let numsMult5 = []
+   for (var i = 0; i < array.length; i++) {
+     numsMult5.push (array[i] * 5)
+  } return numsMult5
+}
+console.log(mult5(myNumbers1));
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
-
+// declare a function mult5two taht takes an arrray as an argument
+const mult5two = (array) => {
+  // declare a local variable array nums2mult5
+  let nums2mult5 = array.map(value => {
+    // map through array and multpile values by 5
+      return value * 5
+  })
+  // return new values
+ return nums2mult5
+}
```

Comment:
good work! altough you can save yourself some time here. you have all of the right logic but you can also return the mapped values outright. here's how you might re-factor:

```javascript
const mult5two = (array) => {
  return array.map(value => value * 5)
}

console.log(mult5two(myNumbers2));
```